src/components/points/Points.tsx

```diff
@@ -1,34 +1,70 @@
-import React, { useState, useEffect } from "react";
+import React, { useEffect } from "react";
 
 import { PointsWrapper } from "../points/PointsElements";
 
-import { DrawedCard } from "../../types/global";
 import pointsHelpers from "./PointsHelper";
 
 import useGameContext from "../../hooks/useGameContext";
 
 function Points({ player = false }) {
-  const { playerCards, croupierCards, isCroupierCardReversed } = useGameContext();
-
-  const [points, setPoints] = useState(0);
-
-  let cards: DrawedCard[];
-
-  if (player) {
-    cards = playerCards;
-  } else {
-    cards = croupierCards;
-  }
+  const {
+    playerCards,
+    croupierCards,
+    isCroupierCardReversed,
+    setMessage,
+    setRoundEnded,
+    points,
+    setPoints,
+    passRound,
+  } = useGameContext();
 
   useEffect(() => {
-    if (cards.length) {
-      let sumPoints = pointsHelpers.getPointsOutcomes(cards, player, isCroupierCardReversed);
+    if ((player && playerCards.length) || (!player && croupierCards.length)) {
```

Comment:
Nie lepiej rozbić to na 2 osobne useeffecty dla gracza i krupiera? Zastanawia mnie też czy to na pewno useeffect powinien nasłuchiwać i ustawiać state, czy to nie powinny byc po prostu funkcje wywoływane na jakiś event (musiałbym dokładniej przeanalizować samą logikę jaką tu wymyśliłeś)